Classes_Homework_5/fromfile.py

```diff
@@ -0,0 +1,181 @@
+import sys
+import fileinput
+import re
+from os import remove
+
+import pyinputplus as pyip
+
+import modules.Functions_Strings_Homework4 as s
+from modules.file import Files
+from modules.combine import Combine
+from modules.dates import Dates
+from modules.input import DateInput
+from modules.exceptions import NoSectionsError, NoValue, PastDate
+from exec_utils.configloader import Config
+
+cnf = Config()
+f = Files()
+d = Dates()
+di = DateInput(cnf.get_values("ERRORS", "past_date"),
+               cnf.get_values("PATTERNS", "date_format"),
+               d.get_current_date())
+decor, decor_length = cnf.get_values("RESTRICTIONS", "n"), \
+                      cnf.get_values("RESTRICTIONS", "count_n")
+# property names (the same as in JSON)
+section, city, text, date, calories = cnf.get_values("PROPERTIES", "label"), \
+                                      cnf.get_values("PROPERTIES", "city"), \
+                                      cnf.get_values("PROPERTIES", "text"), \
+                                      cnf.get_values("PROPERTIES", "date"), \
+                                      cnf.get_values("PROPERTIES", "kcal")
+# names of sections (the same as for manual input and JSON)
+label1, label2, label3 = cnf.get_values("LABELS", "news_label"), \
+                         cnf.get_values("LABELS", "ad_label"), \
+                         cnf.get_values("LABELS", "recipe_label")
+
+
+class WriteFromFile:
+    def __init__(self, default_source, target_file):
+        self.def_source = f.get_path(default_source)
+        self.target = target_file
+
+    def get_raw_source(self):
+        """Get source path(s) from arguments passed, if none - use default"""
+        files = sys.argv[1:]
```

Comment:
For production projects it'd be better to try to use **ArgumentParser()** object from argparse module in order to automatically generate help and usage messages and issue errors when users give the program invalid arguments.

More details: https://habr.com/ru/company/ruvds/blog/440654/